packages/collector/test/tracing/database/mongodb/test.js

```diff
@@ -20,10 +20,12 @@ const globalAgent = require('../../../globalAgent');
 
 const USE_ATLAS = process.env.USE_ATLAS === 'true';
 
-['latest', 'v4', 'v3'].forEach(version => {
+['latest', 'v5', 'v4'].forEach(version => {
   let mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
 
   if (version === 'latest') {
+    mochaSuiteFn = semver.lt(process.versions.node, '16.20.1') ? describe.skip : mochaSuiteFn;
```

Comment:
TBD: Which Node v16 version do we run on CI?

Reply:
https://github.com/instana/nodejs/blob/c193b291958243f3974aff1eac787141dd8e4770/.circleci/config.yml#L444

Reply:
CI runs `v16.20.2` already. Then we are okay with this check. How about we add a comment why we are having such a specific check `semver.lt(process.versions.node, '16.20.1')`? I really never saw this before.

https://github.com/mongodb/node-mongodb-native/blob/main/package.json#L118

Reply:
done